static/js/pages/SiteCreationPage.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react"
```

Comment:
This is just what used to be SiteDashboard.tsx. No changes were made here except for the renamed function